_posts/2025-10-14-geoserver-2-28-0-released.md

```diff
@@ -0,0 +1,220 @@
+* [GEOS-11862](https://osgeo-org.atlassian.net/browse/GEOS-11862) Layer Preview and Tile Layers page dropdown links broken after updating table
+* [GEOS-11863](https://osgeo-org.atlassian.net/browse/GEOS-11863) Use GetTile or GetMap requests in <map-tile src> for cached raster coverages where appropriate
+* [GEOS-11865](https://osgeo-org.atlassian.net/browse/GEOS-11865) MapDownloadProcess washes out 1 band gray images when transparency is on
+* [GEOS-11866](https://osgeo-org.atlassian.net/browse/GEOS-11866) Prevent requests setting variables that should only be set by GeoServer
+* [GEOS-11879](https://osgeo-org.atlassian.net/browse/GEOS-11879) Xalan causes a java.lang.NoClassDefFoundError
+* [GEOS-11902](https://osgeo-org.atlassian.net/browse/GEOS-11902) More compact, easier to maintain conformance configuration UI
+* [GEOS-11910](https://osgeo-org.atlassian.net/browse/GEOS-11910) JMS Cluster settings Section is not showing properly
+* [GEOS-11917](https://osgeo-org.atlassian.net/browse/GEOS-11917) INSPIRE configuration does not get properly saved when OGC API module is included
+* [GEOS-11931](https://osgeo-org.atlassian.net/browse/GEOS-11931) GeoServer binary package fails to start after Jetty 9.x to 10.0.25 upgrade
+
+Task:
+
+* [GEOS-11956](https://osgeo-org.atlassian.net/browse/GEOS-11956) Fix build server WfsCompatibilityTest failure (when testing against "local" GeoServer on port 8080)
+* [GEOS-11813](https://osgeo-org.atlassian.net/browse/GEOS-11813) Create REST API For Security Providers
+* [GEOS-11814](https://osgeo-org.atlassian.net/browse/GEOS-11814) Create a REST API for Filter Chains
+* [GEOS-11815](https://osgeo-org.atlassian.net/browse/GEOS-11815) Create authentication filter REST API
+* [GEOS-11831](https://osgeo-org.atlassian.net/browse/GEOS-11831) OseoDispatcherCallback improvements
+* [GEOS-11852](https://osgeo-org.atlassian.net/browse/GEOS-11852) Remove master password info page
+* [GEOS-11853](https://osgeo-org.atlassian.net/browse/GEOS-11853) Clarify keystore vs master vs root password
+* [GEOS-11854](https://osgeo-org.atlassian.net/browse/GEOS-11854) Generation of security/masterpw.info no longer required
+* [GEOS-11855](https://osgeo-org.atlassian.net/browse/GEOS-11855) global settings cog warning in release data directory
+* [GEOS-11869](https://osgeo-org.atlassian.net/browse/GEOS-11869) Replace entity resolution setting with application property
+* [GEOS-11881](https://osgeo-org.atlassian.net/browse/GEOS-11881) Update postgis-jdbc
+* [GEOS-11882](https://osgeo-org.atlassian.net/browse/GEOS-11882) Cleanup postgis-jdbc dependencies
+* [GEOS-11932](https://osgeo-org.atlassian.net/browse/GEOS-11932) Upgrade Oracle JDBC driver to Java 17
+
+For the complete list see [2.28.0](https://github.com/geoserver/geoserver/releases/tag/2.28.0) release notes. 
+
+## Community Updates
+
+Community module development:
+
+* [GEOS-11848](https://osgeo-org.atlassian.net/browse/GEOS-11848) Migrate use of JAITools to JAI-Ext
```

Comment:
This one does not feel like "community" to me... core?

Reply:
The ticket was flagged "community" and "wps" in the issue tracker so it showed up this way.